src/Cron_Command.php

```diff
@@ -69,15 +69,18 @@ protected static function get_cron_spawn() {
 		$sslverify     = \WP_CLI\Utils\wp_version_compare( 4.0, '<' );
 		$doing_wp_cron = sprintf( '%.22F', microtime( true ) );
 
-		$cron_request = apply_filters( 'cron_request', array(
-			'url'  => site_url( 'wp-cron.php?doing_wp_cron=' . $doing_wp_cron ),
-			'key'  => $doing_wp_cron,
-			'args' => array(
-				'timeout'   => 3,
-				'blocking'  => true,
-				'sslverify' => apply_filters( 'https_local_ssl_verify', $sslverify )
+		$cron_request = apply_filters(
+			'cron_request', // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedHooknameFound
```

Comment:
Please add a comment stating why this is whitelisted.
```suggestion
			'cron_request', // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedHooknameFound -- Calling native WP hook.
```